lib/utils/logger.rb

```diff
@@ -0,0 +1,30 @@
+require 'logger'
+require 'fileutils'
+module XClarityClient
+
+  class XClarityLogger
+    
+    def initialize(global_log=nil)
+      @log = $lenovo_log ||= $log ||= global_log
```

Comment:
Can you add a comment here indicating that the library is looking for an existing log to add entries to?  I think that is an important point for anyone that may be using this client within the context of ManageIQ or outside it.